frontend/catalyst/oqc/oqc_device.py

```diff
@@ -45,7 +45,7 @@ def get_c_interface():
         """
 
         # TODO: Replace with the oqc shared library
-        return "oqc", get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/libdummy_device.so"
+        return "oqc", get_lib_path("oqc_runtime", "OQC_LIB_DIR") + "/librtd_oqc.so"
```

Comment:
I am getting a linking error related to the question you had yesterday:

```
python: symbol lookup error: frontend/catalyst/oqc/src/build//librtd_oqc.so: undefined symbol: _Z14getPythonMutexv
```

I'll look into it. Do you happen to see this same error?

Reply:
That should be solved with your changes